.github/workflows/ci.yml

```
@@ -0,0 +1,75 @@
+name: CI
+on:
+  - push
+  - pull_request
+
+jobs:
+  Test:
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    name: ${{ matrix.os }} - Atom ${{ matrix.atom_channel }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
+         # - macos-latest
+         # - windows-latest
+        atom_channel: [stable, beta]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: UziTech/action-setup-atom@v1
+        with:
+          channel: ${{ matrix.atom_channel }}
+      - name: Versions
+        run: |
+          apm -v
+      - name: Install APM dependencies
+        run: |
+          apm ci  # uses locked module. use `apm install` for non-locked
+      - name: Run tests 👩🏾‍💻
```

You have to install the `busy-signal` package with `apm`. It works locally because you already have that package installed.

```suggestion
      - name: Install Packages
        run: |
          apm install busy-signal
      - name: Run tests 👩🏾‍💻
```

---

Thanks for the tip! Done in https://github.com/atom-ide-community/atom-ide-datatip/pull/66/commits/408b45d5a0d257046fb11bc8e8bcbba43384038f